### ISSUE TITLE
Array double arrow not aligned correctly

### ISSUE THREAD

**Issue by JimmyAppelt** [WordPress-Coding-Standards/WordPress-Coding-Standards#1445]
Hi there,

Sniff:
- WordPress.Arrays.MultipleStatementAlignment.LongIndexSpaceBeforeDoubleArrow

Versions installed:
- PHPCS 3.3.1
- WPCS 1.0.0
- PHP 7.2.0

Got this warning when array keys are a bit longer.
_Note that using `array()` instead of `[]` gives same result._

### All ok

![phpstorm64_2018-08-08_01-57-16](https://user-images.githubusercontent.com/5420393/43808672-63beb6e4-9aae-11e8-816f-b1047d0b02ae.png)

### Making key's a bit longer (prepending test_)

![phpstorm64_2018-08-08_01-56-58](https://user-images.githubusercontent.com/5420393/43808680-6f6570e6-9aae-11e8-8aa3-aae3d5ca304b.png)
![phpstorm64_2018-08-08_01-56-49](https://user-images.githubusercontent.com/5420393/43808695-7d8f62d0-9aae-11e8-99c4-40be6041008d.png)

### Testcode

```php
// ok.
$array1 = [
	'some_custom_longer_key_single_price_unit'         => 'value',
	'some_custom_longer_key_single_legal_info'         => 'value',
	'some_custom_longer_key_single_delivery_time_info' => 'value',
	'some_custom_longer_key_single_product_units'      => 'value',
];

// Not ok.
$array2 = [
	'test_some_custom_longer_key_single_price_unit'         => 'value',
	'test_some_custom_longer_key_single_legal_info'         => 'value',
	'test_some_custom_longer_key_single_delivery_time_info' => 'value',
	'test_some_custom_longer_key_single_product_units'      => 'value',
];
```

**Comment by ocean90** [comment]
You have to increase the `maxColumn` value if you want to have the alignment for longer lines too. See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/8ecba48a254bf34c22328bdf1324cef409c81ed6/WordPress-Core/ruleset.xml#L86 and https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/81a265a2bee11d2b9a040496792c647aa5c97dca/WordPress/Sniffs/Arrays/MultipleStatementAlignmentSniff.php#L55-L76.

**Comment by jrfnl** [comment]
@JimmyAppelt What @ocean90 said, though you'll find a more user friendly explanation of all the properties which can be set in the wiki: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#array-alignment-maximum-column

**Comment by JimmyAppelt** [comment]
@ocean90 @jrfnl thanks guys! Working fine now with the property increase 🙂
